src/tests/test_filters.cpp

```diff
@@ -152,19 +152,19 @@ class Filter_Tests : public Test
 
          // can't explicitly insert a queue into the pipe because they are implicit
          result.test_throws("pipe error", "Invalid argument Pipe::append: SecureQueue cannot be used",
-                            [&]() { pipe.append(new Botan::SecureQueue); });
+         [&]() { pipe.append(new Botan::SecureQueue); });
          result.test_throws("pipe error", "Invalid argument Pipe::prepend: SecureQueue cannot be used",
-                            [&]() { pipe.prepend(new Botan::SecureQueue); });
+         [&]() { pipe.prepend(new Botan::SecureQueue); });
```

Comment:
Bogus formatting of lambdas here, not sure how to fix this... disable astyle in this region?